en/peripherals/frsky_telemetry.md

```diff
@@ -54,3 +91,10 @@ Ready-made cables (which include the required adapters) are available from:
 UART to S.PORT adapters can be sourced from:
 * SPC: [getfpv.com](http://www.getfpv.com/frsky-smart-port-converter-cable.html), [unmannedtechshop.co.uk](https://www.unmannedtechshop.co.uk/frsky-smart-port-converter-spc/) 
 * FUL-1: [unmannedtech.co.uk](https://www.unmannedtechshop.co.uk/frsky-transmitter-receiver-upgrade-adapter-ful-1/)
+
+
+## Additional Information
+
+For additional information, see the following links:

```

Comment:
@bkueng FYI I just pasted your links here. They may not be best, but I'm not ready to provide better yet.

Reply:
These are fine. Chances are high if you have another transmitter/receiver and you do a specific google search you find a blog post on how to set it up.